lib/client/configurator/configurator_test.go

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
@russjones can you take a look at this alternative design:

https://github.com/cyphar/filepath-securejoin/blob/master/join.go

I've noticed docker is using it. I wonder if we should use it instead of the home-grown one.

Reply:
Looks good to me.